Cargo.toml

```
@@ -15,11 +15,13 @@ default-run = "scd4x-util"
 [features]
 scd41 = []
 util = [ "structopt", "simplelog", "humantime", "linux-embedded-hal", "anyhow" ]
```

```suggestion
util = [ "dep:structopt", "dep:simplelog", "dep:humantime", "dep:linux-embedded-hal", "dep:anyhow" ]
```
Since `dep:` is used for `embedded-hal-async` it now also needs to be used for all other optional dependencies.